### PR TITLE
[web] Improve CRM tour keyboard navigation

### PR DESCRIPTION
**Pull request by Arcasias**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/js/views/kanban/quick_create_form_view.js [hunk 1]
```diff
@@ -80,6 +80,23 @@ var QuickCreateFormController = FormController.extend({
     getChanges: function () {
         return this.model.getChanges(this.handle);
     },
+
+    /**
+     * @override
+     */
+    renderButtons() {/* Buttons are given, not rendered */},
```
Comment (Polymorphe57) [inline addons/web/static/src/js/views/kanban/quick_create_form_view.js:87]:
```suggestion
    renderButtons() {
      this.$buttons = $(qweb.render('KanbanView.RecordQuickCreate.buttons'));
    },
```

File: addons/web_tour/static/tests/tour_manager_tests.js [hunk 2]
```diff
@@ -114,5 +114,35 @@ odoo.define('web_tour.tour_manager_tests', async function (require) {
...
+                tours: [{
+                    name: "Tour",
+                    options: { rainbowMan: false },
+                    steps: [{ trigger: "input.anchor" }],
+                }],
+            });
+
+            assert.containsOnce(document.body, '.o_tooltip:visible');
+
+            $(".anchor").replaceWith($(anchorTemplate));
```
Comment (Polymorphe57) [inline addons/web_tour/static/tests/tour_manager_tests.js:134]: to do: keep same element like in FieldMonetary case

File: addons/crm/static/src/js/tours/crm.js [hunk 3]
```diff
@@ -35,9 +35,10 @@ tour.register('crm_tour', {
     trigger: ".o_kanban_quick_create .o_field_widget[name='name']",
```
Comment (Polymorphe57) [inline addons/crm/static/src/js/tours/crm.js:35]: - Rewrite 'This change allows certain widgets as the FieldMonetary (which keeps the
same input while detaching/reattaching it to the DOM) to keep consistent
handlers.'
-Make clear the link with keynav

**Comment by VincentSchippefilt** [comment]
robodoo delegate=Polymorphe57

**Comment by VincentSchippefilt** [comment]
robodoo rebase-ff

**Comment by Polymorphe57** [comment]
robodoo r+

**Pull request closed without merging**